hw/xbox/nv2a.c

```diff
@@ -1972,6 +2091,30 @@ static uint8_t* convert_texture_data(const TextureShape s,
           }
         }
         return converted_data;
+    } else if (s.color_format
+                   == NV097_SET_TEXTURE_FORMAT_COLOR_SZ_R6G5B5) {
+        uint8_t *converted_data = g_malloc(width * height * 3);
+        int x, y;
+        for (y = 0; y < height; y++) {
+            for (x = 0; x < width; x++) {
+                uint8_t v;
+                uint16_t rgb565 = *(uint16_t*)(data + y * pitch + x * 2);
+                int8_t *pixel = &converted_data[(y * width + x) * 3];
+                /* FIXME: This is pretty ugly code to extend from
+                 * signed 565 to signed 888 (hopefully)
+                 */
+                v = (rgb565 & 0xF800) >> 8;
+                pixel[0] = (v & 0x80) ? *(int8_t*)&v * 16 / 0x80
```

Comment:
A simple oversight! I never noticed it's 655! Will fix that later.
(It's a delta texture so I can't use apitrace to debug it and the Bumpmapping looked okay)
I'll also use your sign extension - I struggled with that but was to emberassed to ask